polarion/extract_from_polarion.py

```diff
@@ -0,0 +1,91 @@
+class polarian_data:
```

Comment:
Can we have top level documentation along with class and method docStrings

Reply:
done